src/Balances.js

```diff
@@ -1,30 +1,27 @@
 import React, { useEffect, useState } from "react";
 import { Table, Grid } from "semantic-ui-react";
 
+import { useSubstrate } from "./substrate";
+
 export default function Balances(props) {
-  const { api, keyring } = props;
+  const { api, keyring } = useSubstrate();
   const accounts = keyring.getPairs();
   const [balances, setBalances] = useState({});
 
   useEffect(() => {
     const addresses = keyring.getPairs().map(account => account.address);
-    let unsubscribeAll;
+    let unsubscribeAll = null;
 
     api.query.balances.freeBalance
       .multi(addresses, currentBalances => {
         const balancesMap = addresses.reduce(
           (acc, address, index) => ({
-            ...acc,
-            [address]: currentBalances[index].toString()
-          }),
-          {}
-        );
+            ...acc, [address]: currentBalances[index].toString()
+          }), {});
         setBalances(balancesMap);
-      })
-      .then(unsub => {
+      }).then(unsub => {
         unsubscribeAll = unsub;
-      })
-      .catch(console.error);
+      }).catch(console.error);
```

Comment:
Are we following some styleguide? It would be nice to know which form is deemed correct for stuff like this so we don't keep changing it back and forth as parts of other PRs. Especially since it can lead to merge conflicts.

Reply:
unfortunately I don't. It is mostly my past habit. Sometimes want to save loc, sometimes want to have less nesting. 

Understood it is arbitrary. Will look at the airbnb guide you mentioned, and adjusted if needed.